ruby/vampires.rb

```diff
@@ -0,0 +1,71 @@
+
+#Employee Total
+puts "How many employees"
+employee_total= gets.chomp.to_i
+#Loop 
+employee=0
+until employee == employee_total
+
+#Name
+puts "what's your name"
+name=gets.chomp.capitalize!
+
+#Age
+puts "how old are you"
+age =gets.chomp.to_i
+
+#Year
+puts "What year were you born"
+year= gets.chomp.to_i
+if 2016 - year == age
```

Comment:
Instead of hardcoding the year, 2016 for the `current_year`, check out the Ruby Date module (check it out in the Ruby docs) and use `Date.now` so your program could be used in any year.